src/sage/rings/tests.py

```diff
@@ -437,23 +437,27 @@ def test_karatsuba_multiplication(base_ring, maxdeg1, maxdeg2,
 
     Test Karatsuba multiplication of polynomials of small degree over some common rings::
 
-        sage: for C in [QQ, ZZ[I], ZZ[I, sqrt(2)], GF(49, 'a'), MatrixSpace(GF(17), 3)]:
-        ....:     sage.rings.tests.test_karatsuba_multiplication(C, 10, 10)
+        sage: for C in [QQ, ZZ[I], GF(49, 'a'), MatrixSpace(GF(17), 3)]:
+        ....:     test_karatsuba_multiplication(C, 10, 10)
 
     Zero-tests over ``QQbar`` are currently very slow, so we test only very small examples::
 
-        sage.rings.tests.test_karatsuba_multiplication(QQbar, 3, 3, numtests=2)
+        test_karatsuba_multiplication(QQbar, 3, 3, numtests=2)
```

Comment:
The `sage:` prompt seems to be missing here

Reply:
I missed this, will fix.